lock/client/client.go

```diff
@@ -102,6 +111,10 @@ func (d *Driver) reportMountEndpoint(endpoint string, ut *config.UseMount) error
 		return errored.Errorf("Status was not 200: was %d: %q", resp.StatusCode, strings.TrimSpace(string(content)))
 	}
 
+	if resp.StatusCode == 404 {
```

Comment:
I think this code is unreachable as we would return earlier due to the check at line 110